src/pages/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+            /////////////////////////////////
+            //REDIRECT TO SIGN UP PAGE HERE//
+            /////////////////////////////////
+            console.log('REDIRECT TO SIGN UP PAGE')
+            return 0;
+        }
+        e.preventDefault();
+
+        setEmailErrorMessage('');
+        setPasswordErrorMessage('');
+        setEmailErrorVisibility('hidden');
+        setPasswordErrorVisibility('hidden');
+
+        // Validations
+
+        let emailIsValid = true;
+        let passwordIsValid = true;
+
+        let emailHasAnAt = false;
+        let emailHasADomain = true;
+        currentEmail.split('').forEach((char, index, arr) => {
+            if(char === '@'){
+                emailHasAnAt = true;
+            }
+            if(arr.indexOf('@') === arr.length - 1){
+                emailHasADomain = false;
+            }
+        })
+        if(currentEmail === ''){
+            emailIsValid = false;
+            setEmailErrorMessage('Please enter your email');
+            setEmailErrorVisibility('visible');
+        } else if(!emailHasAnAt){
+            emailIsValid = false;
+            setEmailErrorMessage('Invalid email, please add an @ to the email');
+            setEmailErrorVisibility('visible');
+        } else if(!emailHasADomain){
+            emailIsValid = false;
+            setEmailErrorMessage('Invalid email, add a domain next to the @');
+            setEmailErrorVisibility('visible');
+        }
+
+        if(currentPassword.length < 4){
+            passwordIsValid = false;
+            if(currentPassword === ''){
+                setPasswordErrorMessage('Please enter your password');
+                setPasswordErrorVisibility('visible');
+            } else {
+                setPasswordErrorMessage('Invalid password, the password is too short');
+                setPasswordErrorVisibility('visible');
+            }  
+        }
+
+        if(emailIsValid && passwordIsValid){
+            /////////////////////////////
+            //REDIRECT TO HOMEPAGE HERE//
+            /////////////////////////////
+            console.log('REDIRECT TO HOMEPAGE')
+        }
+
+    }
+
+    const handleChange = e => {
+        if(e.target.name === 'login-email'){
+            setCurrentEmail(e.target.value);
+        } else {
+            setCurrentPassword(e.target.value);
+        }
+    }
+
+    return(
+        <div className="login-form-container">
+            <form className="login-form">
+                <fieldset className="login-login-fieldset">
+                    <h1 className="login-form__legend">Sign In</h1>
+                    <div className="login-input-container">
+                        <div className="input-container__input">
+                            <span className="login-input__icon"><FontAwesomeIcon icon={faUser}/></span><input onChange={handleChange} className="login-input__input" type="email" name="login-email" id="login-email" placeholder="Email" required />
```

Comment:
it's easier to read code when add <input> tag in a new line. For example:

```
<div>
 <span></span>
 <input />
</div>
```

Reply:
Done, I separated the tags